master/static/migrations/20231006105547_update-modified-at-for-certain-columns-on-users-table.tx.up.sql

```diff
@@ -0,0 +1,6 @@
+DROP TRIGGER IF EXISTS autoupdate_users_modified_at ON users;
+
+CREATE TRIGGER autoupdate_users_modified_at
+  BEFORE INSERT OR UPDATE OF password_hash, admin, active, display_name, remote ON public.users
```

Comment:
I need to list columns. Are there better way to trigger function? I couldn't find `except` or `ignore` specific column option in postgres trigger

Reply:
yeah, it'd be great if we could do it the other way around. As is if we add a column it would be easy to miss to add it here.